tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_composite_ops.cpp

```diff
@@ -826,6 +826,22 @@ namespace tt::tt_metal::detail{
                 "output_mem_config", "Layout of tensor in TT Accelerator device memory banks", "MemoryConfig", "Default is interleaved in DRAM", "No"
         )doc");
 
+         m_tensor.def("div_no_nan", &div_no_nan,
+            py::arg("input_a").noconvert(), py::arg("input_b").noconvert(), py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG, R"doc(
+            Performs the element-wise div_no_nan on ``input_a`` and ``input_b``.
```

Comment:
Can you add a line describing that where the values of input_b == 0, then output is 0

Reply:
Sure